core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly

```diff
@@ -8,19 +8,24 @@
               <div class="modal-content">
                 
                 <div class="modal-header">
-                  <h4 class="modal-title">${%Security Token}</h4>
+                  <h4 class="modal-title">${%Getting Started}</h4>
                 </div>
                 <div class="modal-body">
                   <i class="water-mark icon-service"></i>
                   <div class="jumbotron welcome-panel offline">
                   
-                    <h1>${%Security Token}</h1>
-                    <p>${%As an extra measure of security, please enter the setup security token to proceed.}</p>
-                    <p>${%It can be found in the logs for this Jenkins instance.}</p>
+                    <h1>${%Unlock Jenkins}</h1>
+                    <p>${%To ensure Jenkins is securely set up by the administrator, a setup security token has been printed to the logs.}
+                        (<small>
```

Comment:
Putting this in parentheses looks like something that works well in English but may not in some languages.